chaos-days/blog/2024-10-14-realistic-benchmarks/index.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: posts
+title:  "Using realist benchmarks to test performance to derive new cluster configurations"
+date:   2024-10-14
+categories: 
+  - chaos_experiment 
+  - performance
+tags:
+  - benchmark
+authors: rodrigo
+---
+
+# Chaos Day Summary
+
+Our first goal is to further improve and make the benchmarks more 
```

Comment:
:question: What does improve mean in this case? I mean, I know, but not sure the average reader would. So here by improving, we mean what - making it more accurate? Improving our understanding of the performance?

Reply:
I wrote the following: 

> Our first goal is to have a benchmark that better reflects a realistic use 
> case from our clients. To achieve this we used a new process model that resembles real use
> cases that we have seen in the field. This process model is a bank dispute
> process that has several service tasks and multi instances.